staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -570,6 +573,39 @@ func GetFieldManagerFlag(cmd *cobra.Command) string {
 	return GetFlagString(cmd, "field-manager")
 }
 
+type ValidationDirective string
+
+const (
+	ValidationDirectiveStrict ValidationDirective = "Strict"
+	ValidationDirectiveWarn   ValidationDirective = "Warn"
+	ValidationDirectiveIgnore ValidationDirective = "Ignore"
+)
+
+func GetValidationDirective(cmd *cobra.Command) (ValidationDirective, error) {
+	var validateFlag = GetFlagString(cmd, "validate")
```

Comment:
was `--validate` (with no value) valid and equivalent to `--validate=true` previously? if so, make sure we continue honoring that